### PR TITLE
Set empty value by default in properties file

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR sets default values (empty values) in the properties file.

This PR is related to the following PR.
https://github.com/scalar-labs/scalardb/pull/780

In the current properties file, when we don't set an environment variable, the `Apache Commons Text` syntax strings are treated as a pure string value. And, sometimes, it causes an unexpected error as follows.

* Configuration example
  ```properties
  scalar.db.sql.server.port=${env:SCALAR_DB_SQL_SERVER_PORT}
  ```
* Error
  ```console
  2023-04-26 12:23:42 java.lang.IllegalArgumentException: the specified value of 'scalar.db.sql.server.port' is not a number. value: ${env:SCALAR_DB_SQL_SERVER_PORT}
  ```

Also, before, we set the default value as `""` (empty value) in the Go template syntax configuration for the `dockerize` command as follows.
```properties
scalar.db.contact_points={{ default .Env.SCALAR_DB_CONTACT_POINTS "" }}
```

In this configuration, when we don't set environment variable `SCALAR_DB_CONTACT_POINTS`, the `dockerize` command sets `""` as follows.
```properties
scalar.db.contact_points=
```

In this case, ScalarDB can treat the empty value properly (set default value in their internal).

So, I added empty values as default values (`""`) in each property to make the same behavior to avoid unexpected errors and make it more explicit for users.

Please take a look!

### CODE REVIEW

File: server/conf/database.properties [hunk 1]
```diff
@@ -1,29 +1,29 @@
 # Comma separated contact points. For DynamoDB, the region is specified by this parameter.
-scalar.db.contact_points=${env:SCALAR_DB_CONTACT_POINTS}
+scalar.db.contact_points=${env:SCALAR_DB_CONTACT_POINTS:-}
```
[the remark this answers is not in the document]
Comment (kota2and3kan) [reply server/conf/database.properties:2]: Ah, I see. Thank you for pointing this out!
Indeed, these default value configurations are duplicated.
I think we can remove the default env values from Dockerfile.
I will do it!

File: server/Dockerfile [hunk 2]
```diff
@@ -19,15 +19,6 @@ WORKDIR /scalardb
...
 ADD server.tar .
 
-ENV SCALAR_DB_CONTACT_POINTS 'cassandra'
-ENV SCALAR_DB_CONTACT_PORT '9042'
-ENV SCALAR_DB_USERNAME ''
-ENV SCALAR_DB_PASSWORD ''
-ENV SCALAR_DB_STORAGE 'cassandra'
-ENV SCALAR_DB_TRANSACTION_MANAGER 'consensus-commit'
-ENV SCALAR_DB_ISOLATION_LEVEL ''
-ENV SCALAR_DB_CONSENSUSCOMMIT_SERIALIZABLE_STRATEGY ''
```
Comment (kota2and3kan) [inline server/Dockerfile:29]: These env variable configurations only set default values.
ScalarDB can set default values properly if the env variables are empty.
So, we can remove these env variables that set just default values.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by komamitsu — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**